src/index.html

```diff
@@ -287,8 +287,8 @@ <h3>Announcing Flutter 1.22</h3>
         {% asset homepage/news-2.png alt='A news item image' class="card-img-top" %}
         <div class="card-body">
             <p class="card-text"><small class="text-muted">News</small></p>
-            <h3>Announcing Flutter Windows Alpha</h3>
-            <a href="https://medium.com/flutter/announcing-flutter-windows-alpha-33982cd0f433">Read more</a>
+            <h3>Sign up for Flutter Engage on 3/3!</h3>
```

Comment:
Don't want to imply that the signup is 3/3...

```suggestion
            <h3>Sign up for Flutter Engage, happening 3/3!</h3>
```